src/MezzTest.cpp

```diff
@@ -80,14 +80,14 @@ namespace
     {
         CallingTableType CallingTable;
 
-        CallingTable[HelpToken] = [&Results](){ Results.ExitWithError = EXIT_FAILURE; };
-        CallingTable[RunInThisProcessToken] = [&Results]{ Results.InSubProcess = true; };
-        CallingTable[NoThreads] = [&Results]{ Results.ForceSingleThread = true; };
-        CallingTable[JunitXMLAToken] = [&Results]{ Results.EmitJunitXml = true; };
-        CallingTable[JunitXMLBToken] = [&Results]{ Results.EmitJunitXml = true; };
+        CallingTable[HelpToken] = [&Results]() noexcept { Results.ExitWithError = EXIT_FAILURE; };
+        CallingTable[RunInThisProcessToken] = [&Results]() noexcept { Results.InSubProcess = true; };
+        CallingTable[NoThreads] = [&Results]() noexcept { Results.ForceSingleThread = true; };
+        CallingTable[JunitXMLAToken] = [&Results]() noexcept { Results.EmitJunitXml = true; };
+        CallingTable[JunitXMLBToken] = [&Results]() noexcept { Results.EmitJunitXml = true; };
```

Comment:
noexcept lambdas!

I knew it could be done, I just hadn't expected to see it.

Reply:
GCC 10 cares, apparently.